src/components/SingleDatePickerInput.jsx

```diff
@@ -19,6 +19,7 @@ import { ICON_BEFORE_POSITION, ICON_AFTER_POSITION, OPEN_DOWN } from '../constan
 const propTypes = forbidExtraProps({
   ...withStylesPropTypes,
   id: PropTypes.string.isRequired,
+  children: PropTypes.node,
```

Comment:
do we need to allow any kind of children here?

Reply:
I think so, because we parent the daypicker in the range selector (at least) so that it points the arrow up to the input itself